### DISCUSSION TITLE
Fan Speed Automatically Jumps to 100% on Second Layer Despite Settings

### DISCUSSION THREAD

**Discussion by ViniFariaXD** [OrcaSlicer/OrcaSlicer#7216] [Dev]
Hello,

I'm experiencing an issue with fan speed control in Orca Slicer. No matter how I configure the fan settings, the fans always go to 100% on the second layer, even though I set a lower maximum limit. After the second layer, the fan speed reduces to the limit I configured, but the spike to 100% is a problem for me because my fans are too strong to run at full power.

I can't use the fans at 100%, and I need to find a solution to prevent this automatic speed increase on the second layer.

Has anyone else encountered this issue? Any advice or suggestions would be greatly appreciated!

Thank you!

**Comment by ViniFariaXD** [comment]
![1](https://github.com/user-attachments/assets/117159d2-04de-4424-9cd3-055f2d6700ba)

**Reply by Budro12** [reply]
I have a similar problem as well. Have you found a solution to this issue?
